assets/src/js/header.js

```diff
@@ -187,11 +191,10 @@ const setMobileTabsMenuScroll = () => {
       lastScrollDir = dir;
       lastScrollRef = lastScrollTop;
       menu.classList.remove('mobile-menu-hidden');
-      return;
```

Comment:
Why did you remove this one? There are others in the same file (lines 178 & 186) that you didn't touch

Reply:
I have removed all the others, the return statement was not necessary.

Reply:
For performance it would probably better to find a way to keep doing this